README.md

```diff
@@ -6,23 +6,22 @@ Contains the source code for the NativeScript's Android runtime. The android-run
 # Main Projects
 The repo is structured in the following projects (ordered by dependencies):
 
-* **android-metadata-generator** - generates metadata necessary for the Android Runtime.
-* **android-binding-generator** - enables Java & Android types to be dynamically created at runtime. Needed by the `extend` routine.
-* **android-runtime** - contains the core logic behind the NativeScript's Android Runtime. This project contains native C++ code and needs the Android NDK to build properly.
-* **android-runtime-testapp** - this is a vanilla Android Application, which contains the tests for the runtime project.
+* **android-metadata-generator** - generates metadata necessary for the Android Runtime. Located in __android-metadata-generator__.
+* **android-binding-generator** - enables Java & Android types to be dynamically created at runtime. Needed by the `extend` routine. Located in __test-app/runtime-binding-generator__.
```

Comment:
Consider adding permalinks to the master branch of where the projects are located inside the repo for easier navigation.

Reply:
I made the changes in the original PR, so I'm closing this